src/panels/lovelace/cards/hui-thermostat-card.ts

```diff
@@ -487,9 +489,11 @@ export class HuiThermostatCard extends LitElement implements LovelaceCard {
     }
 
     data.entity_id = this._config!.entity;
-
-    await this.hass!.callService("climate", service, data);
-
+    try {
+      await this.hass!.callService("climate", service, data);
+    } catch (err: any) {
+      this._error = err.message;
```

Comment:
This is not used in render?

Reply:
Also this should be lokalized :-)

Reply:
Correct, it's not used in render. But when an exception is thrown and not caught, this method doesn't finish executing and as a result, doesn't revert the UI back to its original state.

Reply:
you can catch the error and swallow? No need to set it as a property?

Reply:
I was following the convention I saw in other cards. Maybe setting the error isn't necessary but I don't think swallowing an exception is a good practice.